Gruntfile.js

```diff
@@ -36,7 +36,9 @@ module.exports = function(grunt) {
             "vendor/underscore-1.8.3.js",
             "vendor/backbone-1.1.2.js",
             "vendor/jquery.dataTables-1.9.4.js",
-            "vendor/mock-ajax.js"
+            "vendor/mock-ajax.js",
+            "vendor/bootstrap-3.1.1-dist/js/bootstrap.js",
```

Comment:
really wanting this (filtering) to be a separate, optional plugin in a backdraft-plugins repo. hate pulling BS in as a dependency.